<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SubjectManage</class>
 <widget class="QWidget" name="SubjectManage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Remove Subjecet</string>
  </property>
  <widget class="QTabWidget" name="tWSubjectManage">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>631</width>
     <height>321</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Add Subject</string>
    </attribute>
    <widget class="QPushButton" name="pBClose">
     <property name="geometry">
      <rect>
       <x>524</x>
       <y>222</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Select Grade :</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cBSelectGradeAdd">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>175</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Enter Subject Name :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lESubjectName">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>170</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pBAddSubject">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>220</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Subject</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Select Class</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cBSelectClass">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>111</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Remove Subject</string>
    </attribute>
    <widget class="QPushButton" name="pBClose_2">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>222</y>
       <width>75</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Select Grade :</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cBSelectGradeRemove">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>51</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>180</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Select Subject :</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cBSelectSubjectRemove">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>180</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pBRemove">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>220</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove Subject</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cBSelectClassToRSub">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>120</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>120</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Select Class</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>cBSelectGradeAdd</tabstop>
  <tabstop>cBSelectClass</tabstop>
  <tabstop>lESubjectName</tabstop>
  <tabstop>pBAddSubject</tabstop>
  <tabstop>pBClose</tabstop>
  <tabstop>cBSelectGradeRemove</tabstop>
  <tabstop>cBSelectClassToRSub</tabstop>
  <tabstop>cBSelectSubjectRemove</tabstop>
  <tabstop>pBRemove</tabstop>
  <tabstop>pBClose_2</tabstop>
  <tabstop>tWSubjectManage</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pBClose</sender>
   <signal>clicked()</signal>
   <receiver>SubjectManage</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>580</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>719</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pBClose_2</sender>
   <signal>clicked()</signal>
   <receiver>SubjectManage</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>583</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>665</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
